### ISSUE TITLE
copiapop - Só está a listar os items da primeira página

### ISSUE THREAD

**Issue by vdutch** [fightnight/fightnight-addons#70]
Quando entro na minha conta do copiapop através do addon a listagem dos ficheiros não aparece completa, apenas é listada a primeira página. Experimentei com outras contas e acontece o mesmo.

Se precisares de mais alguma informação é só pedir ;)

**Comment by pedrofscaetano** [comment]
Sim de facto só funciona a 1a pagina

**Comment by lmsr** [comment]
Sim, aqui também, independentemente da plataforma...

**Comment by pedrofscaetano** [comment]
É necessário uma atualização do addon

**Comment by joao35** [comment]
Boa noite,primeiro parabens por terem criado o addon do copiapop e depois quero pedir ao Criador do addon Copiapop(ex Abelhas) para corrigir a maneira de ver as series/filmes que estão nas contas,isto é,entro na minha conta vou a uma serie e depois não aparece no addon os episódios todos dessa serie e penso que seja por faltar a opção próxima pagina,e também há o erro das legendas que não aparecem nas series/filmes que tenho na minha conta e nas outras contas mesmo estando com o mesmo nome do episódio ou filme. Por favor corrigam esses dois erros,Cumps

**Comment by fightnight** [comment]
Testem a nova versão sff

**Comment by pedrofscaetano** [comment]
Confirmo que já passa a página! Muito fixe! 
Falta só exprimentar o OkGoals no Kodi na Apple TV segundo as dicas de um outro user! Mas este está bom

**Comment by fightnight** [comment]
Qual é mesmo o problema? Ainda não consegui compreender :question:

**Comment by asvf** [comment]
Boa noite,

Não consigo mudar de pagina. Dá-me sempre 24 itens da primeira página.
Há algum comando para paginar?

Obrigado.


André

No dia 22/04/2016, às 23:26, pedrofscaetano <notifications@github.com> escreveu:

Confirmo que já passa a página! Muito fixe! 
Falta só exprimentar o OkGoals no Kodi na Apple TV segundo as dicas de um outro user! Mas este está bom

**Comment by fightnight** [comment]
Verifica se tens a v2.0.2 instalada.

**Comment by asvf** [comment]
Confirmo que tenho.

Pesquisa por bluray no exemplo




Obrigado

André

No dia 23/04/2016, às 00:09, fightnight <notifications@github.com> escreveu:

Verifica se tens a v2.0.2 instalada.

**Comment by pedrofscaetano** [comment]
Ok goals - abre o addon, aparecem os jogos, mas dá o erro geral "ok goals error" quando se tentam abrir os resumos dos jogos.
Julgo ser uma questão de o addon não estar adaptado ao kodi na Apple TV! Há outros addons, embora não mts, que acontece o mesmo

**Comment by fightnight** [comment]
Puxei update. Testa agora ;) v1.0.4

**Comment by asvf** [comment]
Confirmam que o copiapop está ok em paginação?
Estarei eu a fazer algo de errado?

Obrigado 


André

No dia 23/04/2016, às 00:28, fightnight <notifications@github.com> escreveu:

Puxei update. Testa agora ;) v1.0.4

**Comment by fightnight** [comment]
@asvf verifica se a versão que tens instalada é a v2.0.2

**Comment by pedrofscaetano** [comment]
Fizeste update do Ok goals?

**Comment by vdutch** [comment]
O passar de página no copiapop já está impecável, obrigado ;)

**Comment by fightnight** [comment]
Sim @pedrofscaetano testa lá

**Comment by lmsr** [comment]
Excelente! Funciona na perfeição! Bom trabalho :)

**Comment by joao35** [comment]
Boa tarde, funciona muito bem,obrigado pela atualização.Cumps

**Comment by asvf** [comment]
Boa tarde,
Tenho a versão 2.02 instalada, no entanto, ao entrar no add-on, pesquisa, vídeo, (ex da pesquisa: 1080p), dá-me no máximo 24 itens por página e só me dá a primeira página, não me aparecendo opção para passar para a segunda.

Pergunta: basta zipar a sub-pasta plugin.video.abelhas e adicionar ao kodi ou é necessário mais alguma coisa para instalar?

Há alguma forma de mudar de página que possa desconhecer?

Muito obrigado,

**Comment by asvf** [comment]
Obrigado pela atenção.
Já instalei do repositório como indicado e continuo no máximo a ver 24 hits nas pesquisas.

Será de ter o kodi 16.0 versão Android box numa MX8?

Obrigado,


André

No dia 23/04/2016, às 22:25, fightnight <notifications@github.com> escreveu:

@asvf instalas o repositório, e depois actualizas automaticamente pelo repositório.

**Comment by LusoWolf** [comment]
Ola, tenho a versão 2.02 do copiapop instalada, mas quando pesquiso so me aparece uma pagina. Esta e a ultima versao? Ja tentei de tudo e nao consigo solucao. A mulher agora sem as novelas fica furiosa!lol  Como era o abelhas era porreiro que tabto para os desenhos animados como as novelas aparecia todos os capitulos e agora nao. Obrigado desde ja.

**Comment by asvf** [comment]
Bom dia,

A mim também me acontece, já testei com uma caixa Android MX8 igual formatada e acontece o mesmo.
Não consigo explicação.

Obrigado


André

No dia 07/05/2016, às 03:06, LusoWolf <notifications@github.com> escreveu:

Ola, tenho a versão 2.02 do copiapop instalada, mas quando pesquiso so me aparece uma pagina. Esta e a ultima versao? Ja tentei de tudo e nao consigo solucao. A mulher agora sem as novelas fica furiosa!lol Como era o abelhas era porreiro que tabto para os desenhos animados como as novelas aparecia todos os capitulos e agora nao. Obrigado desde ja.

**Comment by joao35** [comment]
Boa Tarde, Quero perguntar ao criador do copiapop addon se não pode fazer um addon para o minhateca,isto é, minhateca addon,seria um addon alternativo ao copiapop e igual ao Abelhas.Será que poderá fazer esse addon o mais rápido possivel? Obrigado Cumps

**Comment by maximilianh75** [comment]
Olá,
Queria saber se é possível com o copiapop addon, executá-lo a partir de um arquivo de STRM assim como o youtube plugin.

obrigado

**Comment by blucas73** [comment]
boa noite se possível atualizar o copiapop deixou de funcionar

**Comment by fightnight** [comment]
Verifique se já está a funcionar na última versão.

**Comment by LusoWolf** [comment]
Boa noite. Onde posso arranjar a ultima versao?

**Comment by blucas73** [comment]
Boa noite fightnight obrigado por teres atualizado o addon ma continua a dar erro eu meti o login que tenho no copiapop e continua a dar erro certificai-me que o login estava activo através do pc espero que consigas descobrir o problema e boas FESTAS para ti e todos teus amigos e familiares.

**Comment by rjmlopes** [comment]
Sim, o copiapop continua a dar erro. Seria possivel activar novamente o plugin do minhateca?

Obrigado
RL.

**Comment by Leinad4Mind** [comment]
Na seguinte linha: success=raw.json()['Type']

Dá o erro seguinte: ValueError: No JSON object could be decoded

**Comment by Leinad4Mind** [comment]
@blucas73 @rjmlopes que versão usam do Kodi onde vos dá erro?

**Comment by blucas73** [comment]
Kodi 16 e kodi 17

**Comment by blucas73** [comment]
boa noite tanto no kodi 16 e 17 dá erro a fazer login isto em rasberry pi com osmc em Windows 7 com kodi 16.1 está a funcionar obrigado espero que consigam resolver o problema.

**Comment by blucas73** [comment]
EM Windows faz login mas não deixa reproduzir os vídeos se for através do browser só deixa ver um bocado do filme não deixa ver o filme todo já uma vez tinham feito o mesmo não deixava ver os filmes completos mas alterarão, espero ter ajudado.
